spec/features/robots_spec.rb

```diff
@@ -8,7 +8,6 @@
   it "should have a sitemap index" do
     expect(page).to have_text('User-agent: *')
     expect(page).to have_text('Disallow: /survey-thanks')
-    expect(page).to have_text('Disallow: /404')
```

Comment:
Let's keep this in here…I think we should still disallow it (even if it doesn't exist in this repo).

Reply:
Can you just hardcode this into the `robots.txt.erb` I think?

```rb
<%# Disallow the 404 page in Gatsby (TODO: Migrate this to Gatsby when that owns the robots.txt) %>
Disallow: /404
```

Reply:
Actually…do we need to disallow all `/nz/404` too?  I can write that up one sec…

Reply:
All resolved if CI passes!